lib/heroku/command/domains.rb

```diff
@@ -13,17 +14,35 @@ class Domains < Base
     #Examples:
     #
     # $ heroku domains
-    # === Domain names for example
-    # example.com
+    # === Development Domain
+    # example.herokuapp.com
+    #
+    # === Custom Domains
+    # Domain Name  CNAME Target
```

Comment:
I see. Sorry was on email. It's to not exclude ALIAS records. Fair enough.
Do we need "Record"? how about just "DNS Target"?

On Friday, May 29, 2015, Ryan Brainard <notifications@github.com> wrote:

> In lib/heroku/command/domains.rb
> <https://github.com/heroku/heroku/pull/1589#discussion_r31336919>:
>
> > @@ -13,17 +14,35 @@ class Domains < Base
> >      #Examples:
> >      #
> >      # $ heroku domains
> > -    # === Domain names for example
> > -    # example.com
> > +    # === Development Domain
> > +    # example.herokuapp.com
> > +    #
> > +    # === Custom Domains
> > +    # Domain Name  CNAME Target
>
> @jesperfj <https://github.com/jesperfj> see the thread above
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/heroku/heroku/pull/1589/files#r31336919>.
>